tests/integration/components/container-query-test.js

```diff
@@ -337,54 +365,58 @@ module('Integration | Component | container-query', function(hooks) {
         </div>
       `);
 
-      assert.dom('[data-test-container-query]')
-        .hasAttribute('data-container-query-small')
-        .doesNotHaveAttribute('data-container-query-medium')
-        .doesNotHaveAttribute('data-container-query-large')
-        .doesNotHaveAttribute('data-container-query-short')
-        .hasAttribute('data-container-query-tall')
-        .hasAttribute('data-container-query-ratio-type-A')
-        .hasAttribute('data-container-query-ratio-type-B')
-        .doesNotHaveAttribute('data-container-query-ratio-type-C');
+      assert.areDataAttributesCorrect({
```

Comment:
It may feel a bit strange to pass `undefined` to say that the data attribute doesn't exist. I passed an empty string to check what value the `<ContainerQuery>` component has set for the data attribute.